### PR TITLE
Include user data

### PR DESCRIPTION
**Pull request by Clare2D**
Preps files for translation, includes survey and real estate data. Some changes to the inputs of the create_interactive_report functions so needs to be merged with the other repo.

depends on:
2DegreesInvesting/create_interactive_report/pull/51
2DegreesInvesting/create_interactive_report/issues/52
#230

### CODE REVIEW

**Comment by Clare2D** [comment]
This is a progression from the other pull request which should be merged first theoretically.

**Comment by Clare2D** [comment]
@cjyetman do you know how to "rebase" this so it only shows the commits that are relevant to this PR? 
I believe from 5ebe66bc886646147e134404e7fc796f89b58900 is the starting point. Or should we just merge all of these commits again?

**Comment by cjyetman** [comment]
meaning, that all of the commits seen here before that are actually included in a previous PR? like, you want to rebase this PR on the basis of another PR?

**Comment by Clare2D** [comment]
Yes, the majority are in the previous PR... for this PR and the other one I create last week.

**Comment by cjyetman** [comment]
so... once the previous PR is merged, I think it should be easy enough to do a "Rebase and merge" here... which (as far as I understand) should rebase this PR based on the current master and then merge only the changes needed after that.

**Comment by Clare2D** [comment]
Exactly! I've merged the previous PR this morning so we can start this process. I've also just commented out the disfunctional tests so the checks now pass.

**Comment by cjyetman** [comment]
which is the previous PR?

**Comment by cjyetman** [comment]
I don't see a PR that was merged this morning in this repo. There is still #175 that hasn't yet been merged. And there's #226 which follows this PR.

Tests commented out allows this PR to pass all tests 🎊

**Comment by Clare2D** [comment]
#217

Yes but when the tests don't make sense ie can never pass, it is one solution

File: web_tool_script_1.R [hunk 1]
```diff
@@ -122,9 +123,9 @@ identify_missing_data(portfolio_total)
 
 audit_file <- create_audit_file(portfolio_total)
 
-# create_audit_chart(audit_file, proc_input_path)
+create_audit_chart(audit_file, proc_input_path)
```
Comment (cjyetman) [inline web_tool_script_1.R:126]: this fails for me. anyone know what this means...
```
> create_audit_chart(audit_file, proc_input_path)
`summarise()` ungrouping output (override with `.groups` argument)
Error in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  : 
  polygon edge not found
In addition: Warning messages:
1: In grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
  no font could be found for family "Clear Sans"
2: In grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
 
 Error in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  : 
  polygon edge not found 
```

File: web_tool_script_3.R [hunk 2]
```diff
@@ -218,26 +214,33 @@ if (file.exists(file.path(proc_input_path, portfolio_name, "overview_portfolio.r
...
 indicies_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_bonds_portfolio.rda"))
-peers_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_equity_portfolio.rda"))
-peers_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_bonds_portfolio.rda"))
+peers_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Peers_equity_results_portfolio.rda"))
+peers_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Peers_bonds_results_portfolio.rda"))
+peers_equity_results_user <- read_rds(file.path(data_location_ext, "Peers_equity_results_portfolio_ind.rda"))
+peers_bonds_results_user <- read_rds(file.path(data_location_ext, "Peers_bonds_results_portfolio_ind.rda"))
 
+
+translation_list <- readr::read_csv(path(template_path,"translation_list.csv"))
```
Comment (cjyetman) [inline web_tool_script_3.R:223]: this presumably depends on 2DegreesInvesting/create_interactive_report/pull/51

**Comment by cjyetman** [comment]
my 🎊 was a mini celebration that the it passes now (with the tests commented out), not a criticism 😄

**Review by cjyetman — APPROVED** [review]
I'm approving this having just met with @Clare2D and @jacobvjk and we all understand that this plus another pending PR at PACTA_analysis plus some at StressTest plus a couple PRs that have not even been created yet will be necessary to get everything working together. We hope to get all of that sorted out today.

**Pull request merged**